src/index.html

```diff
@@ -3,11 +3,261 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>The MET</title>
+    <link
+      rel="shortcut icon"
+      href="./images/photos/THEMET.png"
+      type="image/png">
+    <!-- <link rel="stylesheet" href="./styles/main.scss"> -->
+    <link rel="stylesheet" href="/dist/css/main.css">
     <script src="scripts/main.js" defer></script>
   </head>
   <body>
-    <h1>Miami</h1>
+    <h1 hidden>The MET</h1>
```

Comment:
as discussed - no need to a hidden h1 since a relevant h1 is already at the page